app/src/main/java/com/teamagam/gimelgimel/app/network/services/GGMessagingUtils.java

```diff
@@ -111,7 +110,7 @@ public void onFailure(Call<Message> call, Throwable t) {
         try {
             //Synchronous execution of remote API call
             messages = messagesCall.execute().body();
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
commented on another remark.
These are leftover from my refactored. forgot to delete them, did that now !fixed